api/proto/printable.proto

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2018-2020 MobileCoin Inc.
+
+/// Protos to be used for displaying encoded strings to users
+
+syntax = "proto3";
+import "external.proto";
+
+package printable;
+
+option java_package = "com.mobilecoin.api";
+option java_outer_classname = "MobileCoinAPI";
+
+message PaymentRequest {
```

Comment:
this duplicates `GetRequestCodeRequest` from the mobilecoind proto @ https://github.com/mobilecoinofficial/mobilecoin/blob/master/mobilecoind/api/proto/mobilecoind_api.proto#L330

I think it would be better to import from there...

Reply:
The opposite. mobilecoind should move to using the stuff from this PR.